packages/account/src/Components/forms/scroll-to-field-with-error.tsx

```diff
@@ -27,7 +27,7 @@ const ScrollToFieldWithError = ({
     }, [should_recollect_inputs_names]);
     React.useEffect(() => {
         const current_error_field_name =
-            all_page_inputs_names.find(input_name => Object.hasOwn(errors, input_name)) || '';
+            all_page_inputs_names.find(input_name => Object.prototype.hasOwnProperty.call(errors, input_name)) || '';
```

Comment:
why did we call `hasOwnProperty` on prototype and not Object directly? Is it because we want to use .call() ? 🙏

Reply:
there is no property just for Object. this method is in prototype.
we could use errors.hasOwnProperty, but it is not the same functionality, because if there is no such property prototype will be checked. so we should bind the object for checking it only for this object as .hasOwn does.